package.json

```diff
@@ -5,14 +5,15 @@
   "license": "AGPL-3.0",
   "engines": {
     "node": "20.11.0",
-    "npm": "9.9.2"
+    "pnpm": ">=8.0.0"
   },
-  "packageManager": "npm@9.9.2",
+  "packageManager": "pnpm@8.15.0",
```

Comment:
How do make sure that `pnpm` itself gets updated if there are issues with this version if it's pinned here?

Reply:
That is a good question. We typically gravitate towards pinning versions but do we want to pin this at the patch or minor?

Reply:
Do you have a sense of how often breaking changes happen?  With npm, even major version changes were largely compatible with each other.  Pin minor is probably fine, right?

Reply:
okay, updated to only care about minor version. [pin pnpm to minor version](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9476/commits/55dcd3b67bb133b255fbfacde4d31f7b69d60fc6)